docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -693,7 +701,8 @@ import Slider from '@mui/material/Slider';
 export default function App() {
   return (
     <div>
-      <Slider defaultValue={30} className="color-indigo-500" />
```

Comment:
`color-indigo-500` this doesn't exist.

Reply:
Ooops, it should have been `text-indigo-500` I guess.